Tests/AtributikaTests/AtributikaTests.swift

```diff
@@ -709,6 +709,11 @@ class AtributikaTests: XCTestCase {
         XCTAssertEqual(string, test)
         XCTAssertEqual(tags.count, 0)
     }
+    
+    func testAmpersandAndLessThanSymbolCharacterWithDiacritic() {
```

Comment:
`sut should not encounter infinity loop when using ampersand or less than symbol character with diacritic`